generation/3d_autoencoderkl/3d_autoencoderkl_tutorial.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # root_dir = tempfile.mkdtemp() if directory is None else directory
</pre><p>Need to uncomment this line, remove the previous, and import tempfile in the imports section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>